python/graphstorm/config/argument.py

```diff
@@ -1215,21 +1218,98 @@ def input_activate(self):
 
     @property
     def edge_feat_name(self):
-        """ User defined edge feature names. Not be impplemented in this version, but
-            reserved for future usage.
+        """ User provided edge feature names. Default is None.
+
+        .. versionchanged:: 0.4.0
+            The ``edge_feat_name`` property is supported.
+
+        It can be in the following formats:
+
+        - ``feat_name``: global feature name for all edge types, i.e., for any edge, its
+          corresponding feature name is <feat_name>.
+        - ``"can_etype0:feat0","can_etype1:feat0,feat1",...``: different edge types have
```

Comment:
Why can_etype0 ? Why not directly use etype0?

Reply:
updated.